contracts/contracts/FleekERC721.sol

```diff
@@ -624,13 +623,13 @@ contract FleekERC721 is Initializable, ERC721Upgradeable, FleekAccessControl, Fl
      * Requirements:
      *
      * - the AP must exist.
-     * - the sender must have the token controller role.
+     * - the sender must have collection verifier role.
      *
      */
     function setAccessPointNameVerify(
         string memory apName,
         bool verified
-    ) public requireAP(apName) requireTokenRole(_accessPoints[apName].tokenId, TokenRoles.Controller) {
+    ) public requireAP(apName) requireCollectionRole(CollectionRoles.Verifier) {
```

Comment:
I created this card so we can add the verified field to the main NFA and it's setter would also need a require like this.https://app.shortcut.com/fleekxyz/story/657/contracts-add-verified-field-and-allow-verifier-role-to-edit-it